src/background/run.ts

```diff
@@ -30,18 +30,27 @@ export function run(backgroundWindow: Window, browser: typeof global.browser, ch
       }
     }
 
-    // Take a screenshot if no screenshots currently present for the current feedback target
-    if (!target.feedbackState.screenshots.length) {
+    // Take a screenshot if no images currently present for the current feedback target
+    if (!target.feedbackState.images.length) {
       takeScreenshot(target, browser.tabs, store.dispatchers)
     }
   })
 
   store.on('clickTakeScreenshot', state => {
-    takeScreenshot(ensureActiveFeedbackTarget(state), browser.tabs, store.dispatchers)
+    const target = ensureActiveFeedbackTarget(state)
+    if (addImageDisabled(target)) return
```

Comment:
If I'm not mistaken, the button will be disabled so this is redundant.

Reply:
good call, lets' remove those

Reply:
Fixed in latest checkin.